### ISSUE TITLE
Crawling Websites with Infinite Scroll

### ISSUE THREAD

**Issue by n-ohbayashi** [Norconex/crawlers#1077]
Hi.

We are conducting website crawling using NORCONEX.
As more websites adopt infinite scroll, we are looking for ways to crawl such sites.

For example, would it be possible to use a web driver like Selenium to scroll down to the bottom of the page before crawling?
Do you have any suggestions for effective methods?

Thanks.

**Comment by essiembre** [comment]
Yes, using WebDriver would be the way to go. See [WebDriverHttpFetcher](https://opensource.norconex.com/crawlers/web/v3/apidocs/com/norconex/collector/http/fetch/impl/webdriver/WebDriverHttpFetcher.html#nx-xml-usage-heading)

You may have to inject a script after the page loads to constantly simulate user scrolling.  Alternatively, if you find out what backend call is being made, you can also invoke it directly with different parameters each time.

**Comment by n-ohbayashi** [comment]
Thank you for your response. I will consider using WebDriverHttpFetcher.
I will close this issue.

Thank you.
